website/server/controllers/api-v3/chat.js

```diff
@@ -300,6 +300,52 @@ api.clearChatFlags = {
       {$set: {'chat.$.flagCount': message.flagCount}}
     );
 
+    let adminEmailContent;
+    if (user.auth.local) {
+      adminEmailContent = user.auth.local.email;
+    } else if (user.auth.facebook && user.auth.facebook.emails && user.auth.facebook.emails[0]) {
+      adminEmailContent = user.auth.facebook.emails[0].value;
+    }
+
+    let author = await User.findOne({_id: message.uuid}, {auth: 1});
+
+    let authorEmailContent;
+    if (author.auth.local) {
+      authorEmailContent = author.auth.local.email;
+    } else if (author.auth.facebook && author.auth.facebook.emails && author.auth.facebook.emails[0]) {
+      authorEmailContent = author.auth.facebook.emails[0].value;
+    }
+
+    let groupUrl;
+    if (group._id === TAVERN_ID) {
+      groupUrl = '/#/options/groups/tavern';
+    } else if (group.type === 'guild') {
+      groupUrl = `/#/options/groups/guilds/${group._id}`;
+    } else {
+      groupUrl = 'party';
+    }
+
+    sendTxn(FLAG_REPORT_EMAILS, 'flag-report-to-mods', [
```

Comment:
the email name (`flag-report-to-mods`) should be changed to something like `unflag-report-to-mods`